coolest/template/classes/profiles/light.py

```diff
@@ -135,24 +135,43 @@ class LensedPS(AnalyticalProfile):
     """Surface brightness of a set of point sources after being lensed. 
 
     This profile is described by the following parameters:
-    
-    - 'ra_list': list of coordinates along the x axis
-    - 'dec_list': list of coordinates along the y axis
-    - 'amps': list of amplitudes
+    - 'x_true': the value of the true, unlensed x-axis position of the source
+    - 'y_true': the value of the true, unlensed y-axis position of the source
+    - 'm_true': the value of the true, unlensed magnitude of the source
+    - 'x_lensed': list of coordinates along the x axis of the multiple images
+    - 'y_lensed': list of coordinates along the y axis of the multiple images
+    - 'm_lensed': list of magnitudes of the multiple images
+    - 'flag_contains' ('true','lensed','both'): whether the profile contains only the lensed properties, only the true ones, or both
+    - 'flag_coupled': a boolean indicating whether the true positions of the source are coupled to the lensed ones ('flag_contains' must be set to 'both')
     """
 
     def __init__(self):
-        documentation = "Set of lensed point sources"
+        documentation = "Set of point source and lensed multiple images"
```

Comment:
The profile is for a single (unlensed or lensed) point source. For storing multiple point sources, one would then have multiple of such light profiles.